packages/db/src/search/field-mapping.ts

```diff
@@ -92,6 +92,9 @@ const asyncOperationMapping : { [key: string]: Function } = {
   id: (value?: string) => ({
     id: value,
   }),
+  _id: (value?: string) => ({
```

Comment:
What's the reason for `id` in addition to `_id`?    Are we writing both somehow?

Reply:
_id is ElasticSearch document field, and to continue support this type of query, we map _id to corresponding db field. https://github.com/nasa/cumulus/blob/feature/es-phase-2/packages/api/lib/reconciliationReport.js#L34